code/game/machinery/vending.dm

```diff
@@ -122,6 +127,11 @@
 	var/seconds_electrified = 0 //Shock customers like an airlock.
 	var/shoot_inventory = 0 //Fire items at customers! We're broken!
 
+	var/custom_vendor = FALSE //If it's custom, it can be loaded with stuff as long as it's unlocked.
+	var/locked = TRUE
+	var/datum/money_account/machine_vendor_account //Owner of this vendomat. It's the account of the person who registered it.
```

Comment:
I don't see a reason for machine_vendor_account and earnings_account to be two different vars